src/lib/jsonParse/jsonParse.cpp

```diff
@@ -433,7 +432,7 @@ static std::string jsonParse
     if (out != "OK")
     {
       std::string details = std::string("JSON parse error: '") + out + "'";
-      alarmMgr.badInput(clientIp, details);
+      alarmMgr.badInput(clientIp, details, "");
```

Comment:
Makes sense split the details? (similar as described above)

Reply:
Fixed in f1d444ca5f9c1e526bce85c141febb468e447eb9